### ISSUE TITLE
List of administrators

### ISSUE THREAD

**Issue by KiOui** [KiOui/TOSTI#79]
It might be nice to have a page listing all the administrators, and later on maybe their roles (Voorraadcie, Tosticie, etc.)

**Comment by JobDoesburg** [comment]
In addition to this: 

Automatically make users staff if they are in a certain group, and undo if they are not staff anymore

**Comment by JobDoesburg** [comment]
After #87 this is actually quite easy

**Comment by KiOui** [comment]
Just the is_staff users right?

**Comment by JobDoesburg** [comment]
No I would list all order_venues and players and list the users with can_manage_shift and can_control permissions

**Comment by JobDoesburg** [comment]
Or even better: add a 'display on website' field to `GroupSettings` and then list all members of those `GroupSettings.group`'s
